### ISSUE TITLE
Cannot import rules on new install

### ISSUE THREAD

**Issue by ohnonot** [ukanth/afwall#977]
I just had to upgrade my phone from LineageOS 15.1 to 16.

AFWall+ did not work right away. It recognized my old rules & settings, but I could not connect to the internet at all. I could not find any hint on the internet, and I could not fix it myself. In the end I completely uninstalled it, then reinstalled. Fortunately I had previously exported not only the rules, but also profile settings and preferences, so I can just import them now, right?

Unfortunately not: This feature is only available when you donate and get some key to unlock it.

I can still import the rules only, but the export I made is not compatible with that! I think I should have been told this when I made the export, and not now when it's too late.

This last bit seems a little sneaky; I hope it wasn't done like this on purpose.

**Comment by djagoo** [comment]
Same problem here. When I did the export I got no warning that it would be impossible to import without key. It seems as I don't have access to google AppStore I can't even buy one.

**Comment by landry314** [comment]
yeah, you can only export rules and not settings without paying and that is not clear enough to the user before he might wipe his phone and be stuck.

basically, the export rules and settings option should not be available in the unpaid version. this is really the issue.

i suggest you change the title of this to "Don't allow export of rules and settings to unpaid users"
